ember-headlessui/addon/components/combobox/-option.js

```diff
@@ -25,6 +36,7 @@ export default class ComboboxOptionComponent extends Component {
     if (this.args.disabled) return;
 
     this.args.setSelectedOption(this, e);
```

Comment:
Is `this.args.setSelectedOption(this, e)` still needed? The component now handles sending the new selected values (or value) upwards, so I'm not sure this is still required.

Reply:
Probably still necessary,

As inside this callback there is still some magic which relies on DOM elements. 

I think we can remove this as second step, when DOM elements (modifier) is no longer used.